Doc/library/curses.ascii.rst

```diff
@@ -176,7 +176,7 @@ C library:
 
    Checks for a non-ASCII character (ordinal values 0x80 and above).
 
-These functions accept either integers or strings; when the argument is a
+These functions accept either integers or single-character strings; when the argument is a
```

Comment:
Raising TypeError doesn't mean that the function accepts multi-character strings.

"string" means just string here. Although I think it is worth to add the support of bytes here, this PR changes only the documentation.